zuul.d/project-templates.yaml

```diff
@@ -133,6 +133,27 @@
             required-projects:
               - name: github.com/ansible-collections/ansible.netcommon
 
+- project-template:
+    name: ansible-collections-cisco-nxos
+    check:
+      jobs:
+        - ansible-test-network-integration-nxos-cli-python36:
+            vars:
+              ansible_test_collections: true
+            voting: false
+        - build-ansible-collection:
+            required-projects:
+              - name: github.com/ansible-collections/ansible.netcommon
+    gate:
+      queue: integrated
+      jobs:
+        - ansible-test-network-integration-nxos-cli-python36:
+            vars:
+              ansible_test_collections: true
```

Comment:
this can be removed, as non-voting jobs in gate don't do anything to stop merging